dips/dip-20.md

```diff
@@ -124,44 +123,42 @@ account address (see the implementation of
 and
 [`Diem::register_currency`](https://github.com/diem/diem/blob/master/language/stdlib/modules/doc/Diem.md#function-register_currency)).
 
-## <a name="metadata_spec">Required information for registration of a currency and metadata</a>
+## <a name="metadata_spec">Required Information for Registration of a Currency and Metadata</a>
 
-Every registered currency on-chain contains the information listed below.
+Every registered currency on chain contains the information listed below.
 When a currency is registered, the "Mandatory" parameters must be provided
-at the time of registration, other fields are optional or cannot be set at
+at the time of registration. Other fields are optional or cannot be set at
 registration, e.g., the `total_value` field can only be changed through the
-minting and burning of coins:
+minting and burning of coins.
 
 
 | Name                          | Type                                   | Mutable      | Description                                                                                                                             | Mandatory |
 | ----------------------------- | -------------------------------------- | ------------ | -----------------------------------------------------------------------------------------------------------------------                 | --------  |
-| to_xdx_exchange_rate          | FixedPoint32                           | Yes          | The _rough_ exchange rate from `C` to XDX. Used only for dual attestation threshold and mempool gas price normalization                 | Yes       |
+| to_xdx_exchange_rate          | FixedPoint32                           | Yes          | The _rough_ exchange rate from `C` to XDX. Used only for dual attestation threshold and mempool gas-price normalization.                | Yes       |
```

Comment:
Also, a general comment about this table: A number of the descriptions refer to "C" but there's nothing in the section to define what "C" is. It's pretty obvious from the context, but you could just as well omit it from the descriptions, or perhaps write "the currency" if necessary